app/views/devise/confirmations/new.html.erb

```diff
@@ -1,27 +1,67 @@
 <% title "Confirm your email" %>
 <% proper_email = params[:email] || (resource.pending_reconfirmation? ? resource.unconfirmed_email : resource.email) %>
-<div class="crayons-layout crayons-layout--limited-l align-center mt-9 mb-10 p-3">
+<main id="main-content">
+  <% if FeatureFlag.enabled?(:creator_onboarding) %>
```

Comment:
We will maintain the old onboarding workflow in the `else` statement and hide the new flow under a feature flag until the feature is ready to go live.

Reply:
Under the feature flag we've redesigned the page and added a modal that allows us to resend confirmation instructions.